src/Build/BackEnd/Components/RequestBuilder/TargetBuilder.cs

```diff
@@ -363,6 +363,22 @@ void IRequestBuilderCallback.ExitMSBuildCallbackState()
             _requestBuilderCallback.ExitMSBuildCallbackState();
         }
 
+        /// <summary>
+        /// Requests CPU resources from the scheduler.
+        /// </summary>
+        int IRequestBuilderCallback.RequestCores(object monitorLockObject, int requestedCores, bool waitForCores)
```

Comment:
How can these request / release cores get called on the target builder? May be worth documenting as code comment.